sql/core/src/test/scala/org/apache/spark/sql/errors/QueryCompilationErrorsSuite.scala

```diff
@@ -679,6 +680,44 @@ class QueryCompilationErrorsSuite
       context = ExpectedContext("", "", 7, 13, "CAST(1)")
     )
   }
+
+  test("IDENTIFIER_TOO_MANY_NAME_PARTS: " +
+    "create temp view doesn't support identifiers consisting of more than 2 parts") {
+    checkError(
+      exception = intercept[ParseException] {
+        sql("CREATE TEMPORARY VIEW db_name.schema_name.view_name AS SELECT '1' as test_column")
+      },
+      errorClass = "IDENTIFIER_TOO_MANY_NAME_PARTS",
+      parameters = Map(
+        "quoted" -> "db_name.schema_name.view_name",
```

Comment:
db_name.schema_name.view_name should be quoted by backticks.

Reply:
Done